### PR TITLE
rename: rename AddbAsync to AddAsync

### PR DESCRIPTION
**Pull request by gavin1ee**
rename AddbAsync to AddAsync

### CODE REVIEW

**Review by zhenlei520 — APPROVED** [review]
@doddgu

**Comment by zhenlei520** [comment]
After waiting for Auth to merge

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**
